server/README.md

```diff
@@ -29,8 +29,8 @@ Getting Started
 Running Tests
 -------------
 
-Currently, the only tests are API tests (using [gabbi][gabbi]) and PEP8 checks.
-These can be run using [tox][tox].
```

Comment:
These links can stay as-is (the `[tox]` references the link at the bottom of the document), but this looks good.